core/src/main/java/de/danoeh/antennapod/core/storage/PodDBAdapter.java

```diff
@@ -1031,7 +1032,7 @@ public final Cursor getCompletedMediaCursor(int limit) {
 
         return db.query(TABLE_NAME_FEED_MEDIA, null,
                 KEY_PLAYBACK_COMPLETION_DATE + " > 0", null, null,
-                null, String.format("%s DESC LIMIT %d", KEY_PLAYBACK_COMPLETION_DATE, limit));
+                null, String.format(Locale.US, "%s DESC LIMIT %d", KEY_PLAYBACK_COMPLETION_DATE, limit));
```

Comment:
It was causing

![image](https://user-images.githubusercontent.com/833473/78784096-f5ebbf80-79b9-11ea-92a8-04f550867e54.png)

A bug that was there before my changes but had to fix so can see the view.